lib/siilar/client.rb

```diff
@@ -75,6 +74,10 @@ def base_options
         }
       }
 
+      if requests_timeout
+        options.merge!(timeout: requests_timeout)
```

Comment:
Do we want to distinguish timeouts for reading responses from timeouts for open connections?
The way it's set here would be for both. Otherwise we could set `read_timeout` or `open_timeout`
ref: http://www.rubydoc.info/github/jnunemaker/httparty/HTTParty/ClassMethods#read_timeout-instance_method